src/resources/ResourceSnapshots/ResourceSnapshotsInterfaces.ts

```diff
@@ -73,6 +73,11 @@ export enum ResourceSnapshotSupportedFileTypes {
     JSON = 'JSON',
 }
 
+export enum ResourceSnapshotContentType {
+    PRIMARY = 'PRIMARY',
+    CURRENT_STATE = 'CURRENT_STATE',
```

Comment:
I think we were using camel case normally, but I saw that the ZIP and JSON were in uppercase so I guess its ok!